tests/system/action/poll/test_vote.py

```diff
@@ -577,3 +596,1207 @@ def test_default_vote_weight(self) -> None:
         user = self.get_model("user/1")
         assert user.get("vote_$_ids") == ["113"]
         assert user.get("vote_$113_ids") == [1]
+
+
+class VotePollBaseTestClass(BaseActionTestCase):
```

Comment:
Please integrate your existing tests into these ones, so there are no test duplications and test for a specific thing are in one place.

Reply:
Still todo

Reply:
This don't make sense. Integration the tests is not possible in short time. The tests are in one place (place=file).